types/cannon/index.d.ts

```diff
@@ -845,6 +849,35 @@ declare module CANNON {
 
     }
 
+    export class Trimesh extends Shape {
+
+        vertices: number[]
+        indices: number[]
+        scale: Vec3
+
+        constructor(vertices: number[], indices: number[]);
+
+        updateTree(): void;
+        getTrianglesInAABB(aabb: AABB, result: []): [];
+        setScale(scale: Vec3): void
+        updateNormals(): void;
+        updateEdges(): void;
+        getEdgeVertex(edgeIndex: number, firstOrSecond: number, vertexStore: Vec3): void;
+        getEdgeVector(edgeIndex: number, vectorStore: Vec3): void;
+        static computeNormal(va: Vec3, vb: Vec3, vc: Vec3, target: Vec3): void;
+        getVertex(i: number, out: Vec3): Vec3;
+        getWorldVertex(i: number, pos: Vec3, quat: Quaternion, out: Vec3): Vec3;
+        getTriangleVertices(i: number, a: Vec3, b: Vec3, c: Vec3): void;
+        getNormal(i: number, target: Vec3): Vec3;
+        calculateLocalInertia(mass: number, target: Vec3): Vec3;
+        computeLocalAABB(aabb: Vec3): void;
+        updateAABB(): void;
+        updateBoundingSphereRadius(): number;
+        calculateWorldAABB(pos: Vec3, quat: Quaternion, min: number, max: number): void;
```

Comment:
The `min` and `max` parameters seem to be `Vec3` in the source code: https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/shapes/Trimesh.js#L459-L462

```suggestion
        calculateWorldAABB(pos: Vec3, quat: Quaternion, min: Vec3, max: Vec3): void;
```

Reply:
that works better